package.json

```diff
@@ -28,13 +28,14 @@
     "test3": "vue-demi-switch 3 && jest",
     "prebuild": "npm test",
     "build": "npm run build:vue3 && npm run build:vue2",
-    "build:vue3": "npm run build:es & npm run build:commonjs & npm run build:types",
-    "build:vue2": "npm run build:es2 & npm run build:commonjs2",
+    "build:vue3": "npm run build:es & npm run build:commonjs & npm run build:types3",
+    "build:vue2": "npm run build:es2 & npm run build:commonjs2 & npm run build:types2",
     "build:es": "vue-demi-switch 3 && babel src -d es --extensions .ts,.tsx --delete-dir-on-start",
     "build:commonjs": "cross-env BABEL_ENV=commonjs vue-demi-switch 3 && babel src -d lib --extensions .ts,.tsx --delete-dir-on-start",
     "build:es2": "vue-demi-switch 2 vue2 && babel src -d es2 --extensions .ts,.tsx --delete-dir-on-start",
     "build:commonjs2": "cross-env BABEL_ENV=commonjs vue-demi-switch 2 vue2 && babel src -d lib2 --extensions .ts,.tsx --delete-dir-on-start",
-    "build:types": "vue-demi-switch 3 && tsc & tsc --declarationDir ./es",
+    "build:types3": "vue-demi-switch 3 && tsc & tsc --declarationDir ./es",
+    "build:types2": "vue-demi-switch 2 && tsc & tsc --declarationDir ./es2",
```

Comment:
it should be `vue-demi-switch 2 && tsc  --declarationDir ./lib2 & tsc --declarationDir ./es2`